tests/api/common/test_mark_tasks.py

```diff
@@ -49,7 +49,7 @@ def dagbag():
     from airflow.models.dagbag import DagBag
 
     # Ensure the DAGs we are looking at from the DB are up-to-date
-    non_serialized_dagbag = DagBag(read_dags_from_db=False, include_examples=False)
+    non_serialized_dagbag = DagBag(read_dags_from_db=False, include_examples=True)
```

Comment:
Why set i`nclude_examples=True`? Won't it cost some more time?

Reply:
Some API tests use example DAGs: `example_subdag_operator`, `example_trigger_target_dag` and others.
Before changes in this PR all example serialised DAGs stored into database so we did not have issues with it but after we start cleanup DB before each module we have.

I'd rather to say this is temporary solution as target we need to use DAGs from:
- `tests/dags`
- `tests/dags_corrupted`
- `tests/dags_with_system_exit`

Reply:
Yeah. Those changes are needed .

Reply:
> Also found that if we call DAG.sync_to_db() it no has any affect to database, so I change it to same method in DagBag object, I do not know is it expected or it is some kind of bug.

Not sure about it. But I do not see it to have any noticeable effect on time of running the tests, so let's keep it this way.

Reply:
> Yeah. Those changes are needed .

Echo from the past. I trace back to initial commit from 2017 :rofl: